src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.0.0-preview</Version>
```

Comment:
This does not look like a breaking change. You can simply set it as `19.0.1-preview` instead

Reply:
Also, update the `PackageReleaseNotes`

Reply:
Fixed in second commit.